osu.Game.Rulesets.Osu/Objects/Drawables/DrawableSliderBall.cs

```diff
@@ -207,73 +186,35 @@ public void UpdateProgress(double completionProgress)
             lastPosition = newPos;
         }
 
-        private class FollowCircleContainer : CircularContainer
+        private class FollowReceptor : CircularContainer
```

Comment:
What was the reasoning for this `FollowReceptor` change? It seems unrelated to the actual goal of this pull.

(I also slightly dislike that it scrapes out the slider's ball the way it does, rather than the slider caching the ball explicitly and this component resolving it explicitly, but I suppose it's not *horrible*.)

Reply:
So `FollowReceptor` is separate from the actual follow circles because this pull request is supposed to only affect the animations of legacy skins. But because the old follow areas were tied directly to the follow circles themselves, that would mean legacy skins (with their different animations) would have different follow areas than the default skin. `FollowReceptor` has the same animations as the default skin, so they match perfectly. Legacy skins will be a little off, but that's inevitable if legacy skins and default skin are going to have different animations.